downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,62 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type works in ways similar to existing credential types in a standard format using a YAML or JSON-like definition. For example, a custom credential type can inject an API token for a third-party web service into an environment variable for your rulebook or custom inventory script to consume.
+
+Each credential type displays its own unique configurations in the Input Configuration field and the Injector Configuration field, if applicable. Custom credentials support the following ways of injecting their authentication information:
+
+* Environment variables
+* Ansible extra variables
+* File-based templating, which means generating .ini or .conf files that contain credential values
```

Comment:
@jself-sudoku Currently not supported. @matburt should these be removed from documentation. EDA currently doesn't support env var and file based templating like its documented here.

Reply:
Removing... Thanks.